website/_enemies/forest/hawk.md

```diff
@@ -13,6 +13,6 @@ game_name: enemy_hawk
 sprite_image_name: airidle
 ---
 
-Never used in stock levels.
+A mostly complete entity present in the game files that was not used in the completed game. Can be exposed in the level editor through [Dustmod](http://dustmod.com/).
```

Comment:
I don't like linking externally here. Not because of the site but I'd rather talk about this in another section. We should link to an internal reference of Dustmod when we get one.
```suggestion
A mostly complete entity present in the game files that was not used in the completed game. Can be exposed in the level editor through Dustmod.
```